frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -657,6 +667,18 @@ export class TileViewPage extends React.Component {
     }
   }
 
+  storeFilters(filters) {
+    if (this.props.storeFilterKey && this.props.filterRetentionPreference) {
+      const storeFilters = {};
```

Comment:
category filters could be checked here and could override other filters if necessary.

Reply:
Not sure why we would do that. We don't need to store category filters. And this is just a refactor of storing logic into a function so that I can use it in both `clearFilters` and `updateFilters`.

Reply:
Ok got it.